backend/game/cards/keep_cards/health_manipulation_cards/test_even_bigger.py

```diff
@@ -0,0 +1,28 @@
+from game.cards.keep_cards.health_manipulation_cards.even_bigger import EvenBigger
+from game.values import constants
+
+
+def test_even_bigger_gains_2_max_health_when_purchased(player):
+    EvenBigger().immediate_effect(player, None)
+    assert player.maximum_health == constants.DEFAULT_HEALTH + 2
+
+
+def test_even_bigger_gains_2_current_health_when_purchased(player):
+    EvenBigger().immediate_effect(player, None)
+    assert player.current_health == constants.DEFAULT_HEALTH + 2
+
+
+def test_even_bigger_costs_4_energy():
+    assert EvenBigger().cost == 4
+
+
+def test_even_bigger_gains_losses_2_max_health_when_lost(player):
+    EvenBigger().immediate_effect(player, None)
+    EvenBigger().discard_effect(player, None)
+    assert player.maximum_health == constants.DEFAULT_HEALTH
+
+
+def test_even_bigger_gains_losses_2_current_health_when_lost(player):
```

Comment:
duplicate test of test above. Would like to see test of damaged player not losing health if card is discarded, example player max=12, current health=8 -> discard the card -> player max = 10 current health =8